iModelCore/GeomLibs/geom/test/PolyfaceTest/t_lightweightBuilder.cpp

```diff
@@ -97,7 +97,7 @@ static BuilderWrapperPtr Create(double pointGridSize, double normalGridSize, dou
     }
 char const * Name() override { return "LightWeightMap";}
 void GetDescription(char buffer[2048]) override {
-    sprintf(buffer, "(clustered %d)", m_clusteredSearch ? 1 : 0);
+    snprintf(buffer, sizeof(*buffer), "(clustered %d)", m_clusteredSearch ? 1 : 0);

```

Comment:
thanks, I should have realized that...

@dassaf4 I just switched it to Utf8PrintString since that was just as easy as switching the signature. Let me know what you think.